### DISCUSSION TITLE
Add "Custom" label to Dates in a contact entry

### DISCUSSION THREAD

**Discussion by deeeseee** [FossifyOrg/General-Discussion#684] [Ideas]
When adding a phone number to a contact, it has numerous labels (like Mobile, Home, Other, Custom, etc.). When adding an address to a contact, it also has labels like Home, Work, Other & Custom. However, when adding dates to a contact, there is only Anniversary, Birthday and Other.

Would it be possible to add a label called "Custom" when adding new dates to a contact? This would allow us to add custom life milestones dates, if desired, such as "First job", "Date we met", or even "Death".

Since the feature already exists for Phone numbers and Addresses, I hope it's not too much trouble to add that feature to Dates as well.

Thank you for maintaining and improving these apps!

**Comment by Aga-C** [comment]
Please create a new [feature request](https://github.com/FossifyOrg/Contacts/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for this.

**Reply by deeeseee** [reply]
@Aga-C Thanks for the guidance! It appears a [similar feature request](https://github.com/FossifyOrg/Contacts/issues/26) was closed back in Feb 2024. Is there value in submitting a new feature request for this? (I was hoping it would be simpler because the feature already exists in other categories of the app, but I know nothing is as simple as it seems.)

**Reply by Aga-C** [reply]
That request was closed because the author ignored the feature request form and a similar request appeared. However, that other request is only about adding fields that are already defined in other apps, it's not about adding any custom date.

**Reply by deeeseee** [reply]
Ah, I see. New Feature Request created (https://github.com/FossifyOrg/Contacts/issues/154). Thank you!
